setup.bash

```diff
@@ -11,7 +11,7 @@ then
     libncursesw5-dev libreadline6-dev libdb5.3-dev libgdbm-dev libsqlite3-dev \
```

Comment:
The `if $1` in line one is a bit odd. I think a comment line at the beginning of what this means could be helpful

Reply:
Good ideia

Reply:
resolved